SamplesDashboard/SamplesDashboard/ClientApp/src/views/Home.tsx

```diff
@@ -30,7 +30,7 @@ export default class Home extends React.Component<any> {
                         <Repositories isAuthenticated={true} title={'samples'} />
                     </div>
                 </PivotItem>
-                <PivotItem headerText='Code Projects'>
+                <PivotItem headerText='SDKs'>
```

Comment:
changed this name a while back because the projects being tracked aren't just SDKs.

Reply:
Oh I see. The production one still says SDK's, that's why I changed it. Are you saying that you didn't like "Code Projects" either? Should we come up with a satisfactory name? Maybe "Tools & SDKs"?